ister_gui.py

```diff
@@ -1935,6 +1949,14 @@ def handler(self, config):
             return exc
         return self._action
 
+    def _erase_dup_mount(self, point, part):
+        for idx, chc in enumerate(self.choices):
+            if '{} '.format(point) in chc and '{} '.format(part) not in chc:
```

Comment:
Oh wait is that a post formatted map? Is the original data source updated later and we can't (re)create the formatting without duplicates later on?